addons/binding/org.openhab.binding.hdpowerview/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: Hunter Douglas PowerView Binding
+Bundle-SymbolicName: org.openhab.binding.hdpowerview;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.0.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/jcifs-1.3.18.jar
```

Comment:
You need this lib for a single method: `NbtAddress.getByName` - This is doing netBIOS name lookups, which isn't trivial as it seems.
Ideally, I would want to see such a feature in `org.eclipse.smarthome.io.net` - but as this lib is LGPL, we do not have a chance to get it to Eclipse :-/. How hard would it be to re-implement this feature, do you know?